### DISCUSSION TITLE
Custom tooltips?

### DISCUSSION THREAD

**Discussion by AgentSmith0** [vidstack/player#1044] [Q&A]
Is it possible to use custom tooltips, e.g. by disabling the default vidstack tooltips and creating your own by using the title attribute?

**Comment by mihar-22** [comment]
If you're referring to the Default Layout, we don't have an option for disabling tooltips. I'm open to a PR that exposes a prop on the layout to replace tooltips with `title` but we also need to weigh the pros/cons. Not sure if there's any accessibility concerns with the title attribute. I'd also love to know why you'd like this option. Is there something wrong with the tooltips?

**Reply by AgentSmith0** [reply]
It would be nice to have a consistent look through the application by using own custom tooltips.

**Reply by AgentSmith0** [reply]
Maybe just overriding some CSS will do the job for now.
